app/services/replication_transfer_updater.rb

```diff
@@ -83,7 +83,7 @@ def request_storage
       if params[:fixity_value] == correct_fixity
         record.update(store_requested: true)
       else
-        record.cancel!("fixity_reject")
+        record.cancel!("fixity_reject", params[:cancel_reason_detail])
       end
```

Comment:
Should this method be creating the cancel reason detail, with information about why the fixity values fail to match up?